.github/workflows/run_tests.yml

```diff
@@ -83,6 +83,10 @@ jobs:
           args: check --fix-only
       - name: Run tests with pytest
         run: uv run -p .venv pytest tests/
+      - name: Install optional depencies
+        run: uv sync -p .venv --extra dev --extra mcp
```

Comment:
nonblocking for this PR - should we create another group `full_test` which include all these extra deps?

Reply:
Let me create one quickly